telemetry/definitions/commonDefinitions.json

```diff
@@ -6969,4 +7042,4 @@
             ]
         }
     ]
-}
+}
```

Comment:
looks like your editor might be removing the final EOL. please check your editor/IDE settings

Reply:
Looks like it gets removed by running the validation script, but I can add it back manually